src/training/simple.py

```diff
@@ -36,7 +39,7 @@ def __init__(
     def train(self):
         self.model.train()
 
-        for inputs, target in self.train_dataloader():
+        for inputs, target in self.train_dataloader:
```

Comment:
Why don't we need these braces? Or why did we have them in the first place?....

Reply:
We don't need them because the dataloader is an object that we can iterate through. Why we had them in the first place is another question though, I think that got here from your first template draft, so that's kinda my bad for not proof checking that the code works before committing the previous version.

Reply:
Got it.